deps/eventlog/src/ccel/parser/parsers/platform_config_flags.rs

```diff
@@ -0,0 +1,135 @@
+use crate::ccel::parser::EventDataParser;
+use crate::EventDetails;
+use anyhow::Result;
+use serde::Serialize;
+
+pub struct EvPlatformConfigFlagsParser;
+
+#[derive(Debug, Clone, Serialize)]
+pub struct EventDetail {
+    pub name: String,
+    pub value: Option<String>,
+}
+
+/// Parser for EV_PLATFORM_CONFIG_FLAGS.
+/// This event field contents are manufacturer implementation-specific.
```

Comment:
is there a way to check what is "manufacturer" is? the implementation below assumes there's always the kernel cmdline.